test/fixtures/key/expected.js

```diff
@@ -1,4 +1,10 @@
 "use strict";
 
-elementOpen("div", "1");
+elementOpen("div");
+elementOpen("div", "1", ["key", "1"])
+elementClose("div")
```

Comment:
You'd have to hook into the code generation. I don't think that's exposed at all. cc @sebmck